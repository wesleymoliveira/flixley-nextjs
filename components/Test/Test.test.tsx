import { render, screen } from '@testing-library/react';

import { Test } from './Test';

describe('<Test />', () => {
	it('should render the component', () => {
		const { container } = render(<Test />);

		expect(screen.getByTestId(/test/i)).toBeInTheDocument();

		expect(container.firstChild).toMatchSnapshot();
	});
});
