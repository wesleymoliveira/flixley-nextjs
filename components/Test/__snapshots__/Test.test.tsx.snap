// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Test /> should render the component 1`] = `
<div
  class="emotion-0"
  data-testid="test"
>
  <h1>
    Test
  </h1>
</div>
`;
